extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -7,16 +7,18 @@
  * @package Jetpack
  */
 
-if (
-	( defined( 'IS_WPCOM' ) && IS_WPCOM ) ||
-	class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' )
-) {
-	jetpack_register_block(
-		'jetpack/tiled-gallery',
-		array(
-			'render_callback' => 'jetpack_tiled_gallery_load_block_assets',
-		)
-	);
+class Jetpack_Tiled_Gallery_Block {
```

Comment:
Will prolly need `@since` Jetpack pragma here and for public functions?

Reply:
Sure 👍